src/ping-thread.coffee

```diff
@@ -0,0 +1,7 @@
+module.exports = (robot) ->
+	robot.hear /@(thread|t)\b/i, (context) ->
+		if context.robot.adapter.fetchThreadParticipants?
+			context.robot.adapter.fetchThreadParticipants context, (list) ->
+				list = list.filter (item) -> item not in ['@thread', '@t']
```

Comment:
I don't think this filter is necessary, this would just filter out `@t` or `@thread` users if they exist and have commented in the thread, but that should never be the case?